src/main/java/org/jabref/logic/bibtex/comparator/EntryComparator.java

```diff
@@ -65,10 +66,10 @@ public int compare(BibEntry e1, BibEntry e2) {
         // sorted according to last name.
         if (InternalBibtexFields.getFieldProperties(sortField).contains(FieldProperty.PERSON_NAMES)) {
             if (f1 != null) {
-                f1 = AuthorList.fixAuthorForAlphabetization((String) f1).toLowerCase();
+                f1 = AuthorList.fixAuthorForAlphabetization((String) f1).toLowerCase(Locale.ROOT);
```

Comment:
Hm... Generally, I think ROOT is better than ENGLISH for Strings where we use them internally, e.g. to match files names etc..

However you have a point. If the String is language sensitive it might be better to use the user-specific Locale (i.e., the Locale.getDefault();). However in this case we must be sure that it is not used for any comparisons etc., because this would break stuff if compared to Strings that were manipulated with another Locale.